CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 
 ### Misc. Updates
 
+- **AWSAuthSDK**
+   - Upgrade Facebook SDK to 9.0.0 ([PR #3433](https://github.com/aws-amplify/aws-sdk-ios/pull/3433))
```

Comment:
Please update this to note it as a breaking change, and to provide instructions on the new initialization method. Please also note that support for limited login is forthcoming, but not included in this release.

Reply:
Updated